corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
No huge deal either way, but curious why you would think so. Option 1 will mean, adding the extra script line in all the places the `main.js` is loaded, otherwise pages that use `main.js` will break (not to mention one more file for browser to fetch), whereas option 2 avoids it (though at expense of a little DRY). There's gotta be a better way than these two as well